slider/js/script.js

```diff
@@ -0,0 +1,44 @@
+const carouselSlide = document.querySelector(".carousel-slide");
+const carouselImages = document.querySelectorAll(".carousel-slide img");
+// Buttons
+const prevBtn = document.querySelector(".prev");
+const nextBtn = document.querySelector(".next");
+prevBtn.setAttribute("disabled", "disabled");
+prevBtn.classList.remove("active");
+
+
+let counter = 0;
```

Comment:
Be more specific with variables names - other developers should easily understand your code.

size is a imageWidth
counter is something like activeSliderIndex